### DISCUSSION TITLE
Clearing Fault on master

### DISCUSSION THREAD

**Discussion by tragicfame** [Security-Onion-Solutions/securityonion#8787] [Unsupported Versions]
The master node on my layout is reporting a fault and none of the down-channel nodes are reporting faults. I can verify that all services are up and are functioning normally, so-status reports all OK, and salt \* so.status shows all services on all nodes are OK. Events appear to be ingesting properly, Elastic and Kibana appear to be working ok. 

How does one go about seeing what is triggering the fault signal? Basic troubleshooting doesn't seem to be finding the issue. Soup verifies that the latest versions are installed, calling a high.state doesn't appear to change anything.

**Comment by dougburks** [comment]
Where exactly are you seeing the fault?

What is the output of the following?
```
sudo so-status
echo $?
```

**Reply by tragicfame** [reply]
so-status shows all OK and the echo command returned 0.

**Reply by dougburks** [reply]
I don't see a response to my first question, so I'll ask again:

Where exactly are you seeing the fault?

> so-status shows all OK and the echo command returned 0.

To be clear, did you run `echo $?` immediately after running `sudo so-status` so that it shows the exit status properly?
